src/java.base/share/native/launcher/main.c

```diff
@@ -106,6 +115,14 @@ static void parse_checkpoint(const char *arg) {
             is_checkpoint = 1;
         }
     }
+    if (!is_min_pid_set) {
+        const char *checkpoint_arg = "-XX:CRaCMinPid=";
+        const int len = strlen(checkpoint_arg);
+        if (0 == strncmp(arg, checkpoint_arg, len)) {
+            crac_min_pid = atoi(arg + len);
```

Comment:
You're right, but in this implementation we don't care about errno. It's enough to get 0 as a parsed wrong value, because CRaCMinPid is introduced as JVM option with the min value =1, so in case of wrong value, JVM prints an error message and stops.